src/scalecube-transport/api/ProviderInterface.js

```diff
@@ -0,0 +1,27 @@
+// @flow
```

Comment:
I think it will be better to refactor ProviderInterface to TransportProvider

Reply:
What do you mean by this?